api/IDBDatabase.json

```diff
@@ -165,6 +165,193 @@
           }
         }
       },
+      "abort_event": {
+        "__compat": {
+          "description": "<code>abort</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBDatabase/abort_event",
+          "support": {
+            "chrome": [
+              {
+                "version_added": "24"
+              },
+              {
+                "version_added": "23",
+                "version_removed": "24",
+                "prefix": "webkit"
+              }
+            ],
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": "12"
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": [
+              {
+                "version_added": "16"
+              },
+              {
+                "version_added": "10",
+                "version_removed": "16",
+                "prefix": "moz"
+              }
+            ],
+            "firefox_android": {
+              "version_added": "22"
+            },
+            "ie": {
+              "version_added": "10",
+              "notes": "partial"
+            },
+            "opera": {
+              "version_added": "15"
+            },
+            "opera_android": {
+              "version_added": "22"
+            },
+            "safari": {
+              "version_added": "7"
+            },
+            "safari_ios": {
+              "version_added": "8"
+            },
+            "samsunginternet_android": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        }
+      },
+      "close_event": {
+        "__compat": {
+          "description": "<code>close</code> event",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/IDBDatabase/close_event",
+          "support": {
+            "chrome": {
+              "version_added": "31",
+              "notes": "approx"
+            },
+            "chrome_android": {
+              "version_added": true
```

Comment:
This should be 31.

Reply:
I've also changes https://github.com/mdn/browser-compat-data/pull/3469/files#diff-3ed3a7c9f7fc758c09a4274af4776b76R561 to "31", I assume that's correct?